worker/image-cache.js

```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
   return canvas.toDataURL("image/png");
 };
 
-imageFetch = function(success) {
-  var imageCategories = ["buildings", "nature", "objects", "people", "technology"];
+imageFetch = function(resolutionIndex, photoID, success) {
+  resolutionIndex = parseInt(resolutionIndex) || 0;
```

Comment:
Do we need this? Are there parts of our code that are not sending a value for this? If not, why guard for an undefined value?